main.go

```diff
@@ -299,26 +455,30 @@ func createMilestones(baseURL string, token string, projectID string, milestones
 	return nil
 }
 
-func createMilestoneMap(milestoneAPI []milestoneAPI) map[string]milestone {
+func createMilestoneMap(milestoneAPI []milestoneAPI, api string) map[string]milestone {
 	milestones := map[string]milestone{}
 	for _, v := range milestoneAPI {
 		var m milestone
 		m.DueDate = v.DueDate
 		m.ID = strconv.Itoa(v.ID)
 		m.Title = v.Title
+		if api == "github" {
+			m.State = v.State
+			m.Number = v.Number
+		}
```

Comment:
Why is this necessary only for github ?

Reply:
GitHub uses the "number" field to identify a milestone. When using the "id" field, I would get a 404 error since it was the wrong value. Changing it to the milestone number worked.

Reply:
Ah, gotcha :)